README.md

```diff
@@ -1,27 +1,188 @@
+| `mattermost_version`              | :heavy_multiplication_x: | `4.5.0`      | Install version of mattermost                                                                                                                                                                          |    |    |
+| `mattermost_db_user`              | :heavy_multiplication_x: | `mattermost` | Name of database user                                                                                                                                                                                  |    |    |
+| `mattermost_db_password`          |    :heavy_check_mark:    |              | Password for database user                                                                                                                                                                             |    |    |
+| `mattermost_db_name`              | :heavy_multiplication_x: | `mattermost` | Name of database mattermost should use                                                                                                                                                                 |    |    |
+| `mattermost_user`                 | :heavy_multiplication_x: | `mattermost` | User under which the mattermost process runs                                                                                                                                                           |    |    |
+| `mattermost_enterprise`           | :heavy_multiplication_x: | `false`      | Select if the role should setup a enterprise version or the team version. If you set this variable to `true` you have to place a file named mattermost_licence under `files/{{ inventory_hostname }}/` |    |    |
+| `mattermost_teams`                | :heavy_multiplication_x: |              | Mattermost teams that should be created                                                                                                                                                                |    |    |
+| `mattermost_channels`             | :heavy_multiplication_x: |              | Mattermost Channels to be created                                                                                                                                                                      |    |    |
+| `mattermost_admins`               | :heavy_multiplication_x: |              | Mattermost admin accounts to be created                                                                                                                                                                |    |    |
+| `mattermost_Sql_AtRestEncryptKey` |    :heavy_check_mark:    |              | Encrypt key used for sql. Please generate.                                                                                                                                                             |    |    |
```

Comment:
Why do you capitalize this variable and the next ones? Also, it's SQL, not sql

Reply:
What do you mean with `Please generate`

Reply:
fixed